services/web/server/src/simcore_service_webserver/meta_modeling_handlers.py

```diff
@@ -138,15 +176,25 @@ class ParentMetaProjectRef(BaseModel):
     ref_id: CheckpointID
 
 
-class ProjectIterationAsItem(BaseModel):
+class _BaseModelGet(BaseModel):
     name: str = Field(
         ...,
-        description="Iteration's resource name [AIP-122](https://google.aip.dev/122)",
+        description="Iteration's resource API name",
+        x_mark_resouce_name=True,  # [AIP-122](https://google.aip.dev/122)
```

Comment:
who handles these `x_mark` properties?

Reply:
this is a good question! In short, I would like to automate the generation of API models as much as possible and for that I am injecting ``x-` marks in the metadata of the fields so I can identify  some features (similar to the ``pytest.mark`` concept).

For the moment it is an experiment ... and there are not handler in this PR so I should remove it.

I will create a sample with a prototype of the idea separately and will discuss before adding it to the main repo.